Sources/XCTFluent/Docs.docc/index.md

```diff
@@ -0,0 +1,4 @@
+# ``XCTFluent``
+
+XCTFluent provides XCTest extensions to make it easy to write tests that use Vapor's FluentKit library.
+
```

Comment:
There is an empty line down here. The PR doesn't look clean enough for my approval 🤪 !